### DISCUSSION TITLE
번역 업뎃한거 올리려 하는데

### DISCUSSION THREAD

**Discussion by zilrty** [RimWorldKorea/RMK#912] [질문]
버스에 탑승되있는건 어케함?

그냥 하면 탑승된거 없어지는거 아님?

@RimWorldKorea

**Comment by RimWorldKorea** [comment]
버스에 이미 올라간 모드를 수정하고 싶단거야?

아님 메인 갱신하고 싶은데 버스에 이미 뭐가 있어서 어떻게 할지 모르겠단거야?

**Reply by zilrty** [reply]
후자인데 상관 없나보네

그냥 늘하던데로 하면되지?

**Reply by RimWorldKorea** [reply]
메인에 머지할때 같이 병합되니깐 그때 레이블도 같이 바꿔주면 됨
